src/input/mod.rs

```diff
@@ -1139,7 +1140,93 @@ impl State {
                     );
                 }
             }
-            _ => { /* TODO e.g. tablet or touch events */ }
+            InputEvent::TouchDown { event, .. } => {
+                if let Some(seat) = self.common.seat_with_device(&event.device()).cloned() {
+                    let current_output = seat.active_output();
```

Comment:
I think most compositors just assume, that any touchscreen will be part of the internal screen, so we could match on the DRM connector. I am not aware of any other link, though I would hope `event.device()` could give us a reference to the udev device, which somehow *might* tell us more?

Reply:
Sway has a `map_to_output` configuration option for absolute input devices, which matches `xinput --map-to-output`. Not sure how if determines default mappings.

We could add a heuristic for determining which screen is internal, at least.

Reply:
Yeah we need to make this configurable eventually. I would also love to have the same thing for tablets. (Though more free-form than just assign to output. Assign to region (relative to output) or window would also be nice.